src/libsystemd-network/test-dhcp-client.c

```diff
@@ -31,13 +31,6 @@ static bool verbose = true;
 static int test_fd[2];
 static test_callback_recv_t callback_recv;
 static be32_t xid;
-static sd_event_source *test_hangcheck;
-
-static int test_dhcp_hangcheck(sd_event_source *s, uint64_t usec, void *userdata) {
```

Comment:
This means we lose some test coverage, no? As this should never happen, but we don't notice anymore if a regression makes it happen?

Reply:
No, it is replaced with `sd_event_loop() >= 0`, as if the timeout is detected it returns `-ETIMEDOUT`.